applications/openshift/controller/controller_terminated_pod_gc_threshhold/oval/shared.xml

```diff
@@ -0,0 +1,29 @@
+<def-group>
+  <definition class="compliance" id="controller_terminated_pod_gc_threshhold" version="1">
+    <metadata>
+      <title>Ensure that terminated-pod-gc-threshold is enabled.</title>
+      <affected family="unix">
+        <platform>multi_platform_ocp</platform>
+      </affected>
+      <description>Ensure terminated-pod-gc-threshold is enabled.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion comment="terminated-pod-gc-threshold is configured" test_ref="test_controller_terminated_pod_gc_threshhold" negate="true" />
```

Comment:
Rather than `negate="true"`, I wonder if it makes more sense when reading this OVAL to just check if it is set to true?

Reply:
Not sure. Initial reasoning is that it's true by default, which means no special configuration required. To ensure operators don't have to go around manually specifying default settings, swapped to using negate and failing if ``false`` is present.

Reply:
It would be the same result anyway.

Reply:
Not following. If the OVAL is set to check for ``true``, operators would be forced to go specify that configuration manually (even though it's the default). 

Current OVAL will fail if false is present, vs forcing operators to manually configure default settings in their conf files

Reply:
good point